### PR TITLE
Allow per pixel transparency in javascript platform

### LINKED ISSUE #35951: Per Pixel Transparency in html5
**Godot version:** 3.2 GLES2

**OS/device including version:** Linux 5.4.14-2-MANJARO

**Issue description:**
Hi

Per Pixel Transparency works in html5 export ? 

get_tree().get_root().set_transparent_background(true) in my _ready
Display/Window/Per Pixel Transparency/Allowed => true
Display/Window/Per Pixel Transparency/Enabled => true

On export linux, everythings is good and transparent but in html5 i got a black background !
I change the export template to have `background-color: transparent;` in body html and #canvas 

thx

### PR DESCRIPTION
**Pull request by muiroc**
fixes #35951

### CODE REVIEW

File: platform/javascript/os_javascript.cpp [hunk 1]
```diff
@@ -220,6 +220,16 @@ void OS_JavaScript::get_fullscreen_mode_list(List<VideoMode> *p_list, int p_scre
 	p_list->push_back(OS::VideoMode(screen.width, screen.height, true));
 }
 
+bool OS_JavaScript::get_window_per_pixel_transparency_enabled() const {
+	if (!is_layered_allowed()) return false;
```
Comment (akien-mga) [inline platform/javascript/os_javascript.cpp]: Style-wise, this should be:
```
if (!is_layered_allowed()) {
    return false;
}
```

**Review by bruvzg — COMMENTED** [review]
Web/JavaScript should be added to the supported platform list in `OS` docs: https://github.com/godotengine/godot/blob/master/doc/classes/OS.xml#L963

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.1.

**Pull request merged**